### PR TITLE
Adds support for labels to chronosphere_dashboard

### PR DESCRIPTION
**Pull request by bg451**

### CODE REVIEW

**Comment by bg451** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/chronosphereio/terraform-provider-chronosphere/82?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#82** <a href="https://app.graphite.dev/github/pr/chronosphereio/terraform-provider-chronosphere/82?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#81** <a href="https://app.graphite.dev/github/pr/chronosphereio/terraform-provider-chronosphere/81?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bg451 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+Added:
+* Adds support for labels `chronosphere_dashboard`
```
Comment (bg451) [inline CHANGELOG.md:6]:
```suggestion
* Adds support for labels to `chronosphere_dashboard`
```

Comment (prashantv) [inline CHANGELOG.md:6]:
```suggestion
* Adds support for `labels` in `chronosphere_dashboard`
```

**Review by prashantv — APPROVED** [review]

**Comment by bg451** [comment]
### Merge activity

* **Oct 29, 4:41 PM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/chronosphereio/terraform-provider-chronosphere/82).

**Pull request merged**